src/js/shared/reducers/CandidatesReducer.js

```diff
@@ -0,0 +1,3 @@
+export default function candidatesReducer(state = {}) {
```

Comment:
we should still import the candidates constants and pass them in as the initial state:
```
import { REPUBLICAN, DEMOCRAT } from ../constants/index
const initialState = {
    republican: REPUBLICAN,
    democrat: DEMOCRAT
};
```
Then pass it into the reducer:
` candidatesReducer(state=initialState, action) { ...`